src/components/theme/Navigation/ContextNavigation.jsx

```diff
@@ -46,6 +47,9 @@ function renderNode(node, parentLevel) {
             ''
           )}
         </RouterLink>
+         ) : (
+          <a href={flattenToAppURL(node.href)}>{node.title}</a>
```

Comment:
I wonder if UniversalLink would be a better fit. Though would be fine like this, as well.

Reply:
I changed the <a> tag with <UniversalLink> tag because it gives more options